soc/arm/st_stm32/stm32h5/soc.c

```diff
@@ -57,7 +57,7 @@ static int stm32h5_init(void)
 	/* Disable USB Type-C dead battery pull-down behavior */
 	LL_PWR_DisableUCPDDeadBattery();
 #endif /* PWR_UCPDR_UCPD_DBDIS */
-
+	LL_ICACHE_Disable();
```

Comment:
If disabling ICache is required for ethernet operations, please do temporarily and it within ethernet driver.
Otherwise please remove this line

Reply:
OK i will check that

Reply:
Right without the LL_ICACHE_Disable(); all works with zperf UDP/TCP client/server